perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -90,6 +94,25 @@ def AptInstall(vm):
   _Install(vm)
 
 
+def JujuInstall(vm):
+  if vm.isController is True:
+    if vm.vm_group_specs and 'workers' in vm.vm_group_specs:
+      vm.JujuDeploy('cs:trusty/cassandra',
+                    vm.vm_group_specs['workers'].vm_count)
```

Comment:
How are you guaranteeing that the cassandra units will be deployed to the workers group and not the loader/client?